cmd/support-cron-scheduler/res/configuration.yaml

```diff
@@ -31,3 +31,8 @@ Database:
   Timeout: 5s
   Type: postgres
 
+Retention:
+  Enabled: false
+  Interval: 30s    # Purging interval defines when the database should be rid of records above the high watermark.
```

Comment:
Set the default Enabled to true and Interval to 24h
We set false in core-data and support-notification to prevent the breaking change, but there is no breaking change in this service.
The nature of this service is different from core-data and notifications.  The schedule records should not be generated so quick.  Interval 24h should be a fine setting.